udata/templates/organization/dashboard.html

```diff
@@ -14,10 +14,24 @@
 {% endblock %}
 
 {% block main_content %}
-<dashboard-graphs object-id="{{ org.id }}">
-    {# Load spinner until JS is ready #}
-    <div class="text-center"><span class="fa fa-4x fa-cog fa-spin"></span></div>
-</dashboard-graphs>
+{# Load spinner until JS is ready #}
+<div v-if="false" class="text-center"><span class="fa fa-4x fa-cog fa-spin"></span></div>
```

Comment:
I asked myself the same question, but in this very specific case, the spinner has to be working before JS is loaded so before components are loaded ;)